api/graphql/schema.py

```diff
@@ -292,6 +295,26 @@ class UnitsFilter(AuthFilter, django_filters.FilterSet):
     )
 
 
+class UsersFilter(AuthFilter, django_filters.FilterSet):
+    permission_classes = (
+        (UserPermission,) if not settings.TMP_PERMISSIONS_DISABLED else (AllowAny,)
+    )
+
+    @classmethod
+    def resolve_queryset(
+        cls, connection, iterable, info, args, filtering_args, filterset_class
+    ):
+        queryset = super().resolve_queryset(
+            connection, iterable, info, args, filtering_args, filterset_class
+        )
+
+        if not info.context.user:
+            return None
```

Comment:
None might work as good and at least does not return any users and  `return queryset.none()` would do almost same but in difference will return an empty queryset

Reply:
Fix'd!